### ISSUE TITLE
Service start fails after start/stop/wifi switch

### ISSUE THREAD

**Issue by ignasbernotas** [mysteriumnetwork/node#926]
1. Started service
2. Stopped service
3. Switched wifi
4. Start service
Hangs in starting status and then fails with HTTP 500.

```
2019-04-30T08:22:36.501137 [Debug] [NATS.Receiver] Request '0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7.openvpn.dialog-create' topic has been subscribed to
2019-04-30T08:22:36.5018 [Error] Service serve failed: failed to add NAT forwarding rule: not able to determine outbound ethernet interface
2019-04-30T08:22:36.501897 [Warn] [NATS] Disconnected
2019-04-30T08:22:36.502013 [Info] [discovery] stopping discovery loop..
2019-04-30T08:22:38.183732 [Info] [Mysterium.api] Proposal unregistered for node: 0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7
2019-04-30T08:22:38.183748 [Info] [discovery] Proposal unregistered
2019-04-30T08:22:38.18376 [Info] [Mysterium.api] Proposal registered for node: 0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7 service type: openvpn
2019-04-30T08:22:40.385788 [Info] [Detect Outbound IP] IP detected: 192.168.0.146
// service start called
2019-04-30T08:22:40.683015 [Info] [NATS.DialogWaiter] Connecting to: {nats/v1 {0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7.openvpn [nats://testnet-broker.mysterium.network:4222]}}
2019-04-30T08:22:40.747599 [Info] [Mysterium.api] Total proposals: 25 supported: 25
2019-04-30T08:22:45.202956 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:22:55.189689 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:23:00.485713 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:23:09.975643 [Info] [Detect Outbound IP] IP detected: 97.9.6.1
// service start called
2019-04-30T08:23:10.385993 [Info] [NATS.DialogWaiter] Connecting to: {nats/v1 {0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7.openvpn [nats://testnet-broker.mysterium.network:4222]}}
2019-04-30T08:23:10.440766 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:23:15.352582 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:23:27.870533 [Info] [Detect Outbound IP] IP detected: 97.9.6.1
2019-04-30T08:23:28.105612 [Info] [Mysterium.api] Total proposals: 24 supported: 24
2019-04-30T08:23:28.218408 [Info] [Mysterium.api] Total proposals: 24 supported: 24
// service start called
2019-04-30T08:23:28.275912 [Info] [NATS.DialogWaiter] Connecting to: {nats/v1 {0xe427274f56db8fe8dddba4fc3c696d237cb7f1f7.openvpn [nats://testnet-broker.mysterium.network:4222]}}
```

**Comment by soffokl** [comment]
It's partly related to the https://github.com/mysteriumnetwork/node/issues/726. 
We are detecting public IP and outbound IP only on app start, but it obviously can be changed by many cases.

**Comment by soffokl** [comment]
@ignasbernotas I have checked the case with switching WiFi network. The only error that I have found is cleaning up IP forwarding rules. 
This record exists in the log, we are trying to re-apply outdated rule that is not valid anymore. 

`2019-04-30T08:22:36.5018 [Error] Service serve failed: failed to add NAT forwarding rule: not able to determine outbound ethernet interface`

The https://github.com/mysteriumnetwork/node/pull/1019 fixes this problem. 

I'm not fully sure that this is the only problem that we have related to the issue. Can we recheck if the problem persists after merging this PR?
